extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -295,7 +295,7 @@ export class DatabaseUI extends DisposableObject {
         'codeQLDatabases.chooseDatabaseLgtm',
         this.handleChooseDatabaseLgtm,
         {
-          title: 'Adding database from LGTM. Choose a language from the dropdown, if requested.',
+          title: 'Adding database from LGTM.',
```

Comment:
We can drop the full stop too, since the progress popup message always adds a colon at the end of the title. Same suggestion below.
```suggestion
          title: 'Adding database from LGTM',
```